dashboard/app/views/plc/tasks/edit/_written_assignment_task.html.haml

```diff
@@ -4,8 +4,8 @@
 =form_for(@task, url: plc_task_path(@task)) do |f|
   .field
     = render partial: 'plc/tasks/edit/common_editable', locals: {f: f}
-    = f.label :assignment_description
-    = f.text_area :assignment_description, rows: 10, style: 'width: 100%', value: @task.assignment_description
+    = f.label :level_id
+    = f.select :level_id, options_for_select(FreeResponse.all.pluck(:name, :id), selected: @task.level_id)
```

Comment:
This edit experience will be temporary until we generate these objects from `.script` files.